### PR TITLE
Update Provider SDK

### PR DESCRIPTION
**Pull request by bbernays**
Changes:

- [x] Updated integration tests to run on all PRs
- [x] Slack Webhook in integration test only fires when pr is merged
- [x] Created a `testData` package that all mock tests in all other service packages could use
- [x] Renamed packages functions so that the package name would not be there eg: 
`func AppDaemonSets() *schema.Table` -->`func DaemonSets() *schema.Table`
- [x] Ensure all tables have integration tests
- [x] Ensure renaming was completed consistantly

### CODE REVIEW

**Review by roneli — APPROVED** [review]
LGTM

File: .github/workflows/int_test.yml [hunk 1]
```diff
@@ -64,14 +65,14 @@ jobs:
       - name: Integration Test
         run: |
           touch output.log
-          go test -v -p 20 ./resources/integration_tests | tee output.log
+          go test -v -p 20 -tags=integration ./... | tee output.log
           result_code=${PIPESTATUS[0]}
```
Comment (yevgenypats) [inline .github/workflows/int_test.yml:69]: I think there is no need for the output anymore to detect warnings. We do that on the SDK level (if not that we should add it to the SDK level anyway)

**Review by yevgenypats — CHANGES_REQUESTED** [review]
Looks good. one comment + we need to update to the latest sdk version as we had an issue in the last one (im working atm to tag a new sdk version).

File: client/testing.go [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+	cfg := ``
+
+	providertest.TestResource(t, providertest.ResourceTestCase{
+		Provider: &provider.Provider{
+			Name:    "aws_mock_test_provider",
+			Version: "development",
+			Configure: func(logger hclog.Logger, _ interface{}) (schema.ClientMeta, error) {
+				c := &Client{
+					Log: logging.New(&hclog.LoggerOptions{
+						Level: hclog.Warn,
```
Comment (yevgenypats) [inline client/testing.go:52]: why not reuse the same logger here? I think in test the `providertest.TestResource` will call `Configure` with [`testlogger`](https://github.com/cloudquery/cq-provider-sdk/tree/main/testlog) which will eventually call to `t.Log` and will be useful as go automatically knows to print logs only for tests that fail.

**Review by yevgenypats — CHANGES_REQUESTED** [review]
Few comments and good to go. Also upgrade to cq-provider-sdk v0.6.0

File: .github/workflows/int_test.yml (hunk 1 above)
Comment (bbernays) [reply .github/workflows/int_test.yml]: fixed

File: client/testing.go (hunk 2 above)
Comment (bbernays) [reply client/testing.go]: fixed

**Comment by yevgenypats** [comment]
@bbernays looks good, forgot to mention about the changelog update please as well so I can tag it after I merge

**Pull request merged**